resources/packs/gcp/gcp.lr

```diff
@@ -117,35 +119,35 @@ gcp.resourcemanager.binding {
 }
 
 // GCP Compute Engine
-gcp.compute {
```

Comment:
Can we add an alias or a temporary deprecated implementation that returns the new object:
```
gcp {
  compute() gcp.project.computeService
}
```

Reply:
done